install/kubernetes/helm/istio/charts/mixer/templates/create-custom-resources-job.yaml

```diff
@@ -77,11 +77,9 @@ spec:
       containers:
         - name: hyperkube
           image: "{{ .Values.global.hyperkube.repository }}:{{ .Values.global.hyperkube.tag }}"
-          command:
-            - ./kubectl
-            - apply
-            - -f
-            - /tmp/mixer/custom-resources.yaml
+          command: ["/bin/bash","-c",
+            "sleep 30 && \
```

Comment:
not a fan of this approach.  Can you file a followup issue or PR that revs hyperkube to a later version?